ttnn/cpp/ttnn/operations/moreh/moreh_linear/moreh_linear.cpp

```diff
@@ -0,0 +1,22 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "moreh_linear.hpp"
+// #include "ttnn/cpp/ttnn/deprecated/tt_dnn/op_library/moreh_matmul/moreh_matmul_op.hpp"
+#include "ttnn/cpp/ttnn/operations/moreh/moreh_matmul/moreh_matmul.hpp"
+namespace ttnn::operations::moreh::moreh_linear {
+std::optional<Tensor> MorehLinear::invoke(
+    const Tensor& input,
+    const Tensor& weight,
+    const std::optional<Tensor>& bias,
+    std::optional<Tensor> output,
+    const std::optional<MemoryConfig>& output_mem_config,
```

Comment:
```suggestion
    const std::optional<MemoryConfig>& memory_config,
```

Reply:
resolved in https://github.com/tenstorrent/tt-metal/pull/12812/commits/8dec1ff696bd9948f9366cbee8ec5733a58f2081